test/parallel/test-util-inspect-proxy.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const util = require('util');
+
+const target = {};
+const handler = {
+  get: function() { console.log(arguments); return 5; }
```

Comment:
Should be able to replace it with `util.inspect(arguments[0])` but the
point of the test is that this bit of code is *not* run. I included it here
because it was part of the original bug report. Throwing in here would work
too.
On Apr 29, 2016 6:31 AM, "Colin Ihrig" <notifications@github.com> wrote:

> In test/parallel/test-util-inspect-proxy.js
> <https://github.com/nodejs/node/pull/6465#discussion_r61577734>:
>
> > @@ -0,0 +1,22 @@
> > +'use strict';
> > +
> > +require('../common');
> > +const assert = require('assert');
> > +const util = require('util');
> > +
> > +const target = {};
> > +const handler = {
> > +  get: function() { console.log(arguments); return 5; }
>
> Can you drop the console.log().
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6465/files/3d4ff4311a3cf1ff756413756ffc859a663aba99#r61577734>
>